modules/soc_eq/callback_lp2.php

```diff
@@ -0,0 +1,6 @@
+<?php
+   //Callback procedure for mercedes SoC API LP1 
```

Comment:
Das soll sicher LP2 heißen.
Kann man die callbacks in einem Handler zusammenfassen? Das macht eine Skalierung über x Ladepunkte einfacher.

Reply:
Ja, C&P Fehler. Zusammenfassen muss ich mir anschauen, ob die Callback URL auch andere Parameter noch aufnehmen kann und den Access Code einfach hinten anhängt.Damit wird die Konfiguration allerdings etwas schwieriger. Nach der aktuellen API sind nur 3 Callback URLs möglich, von daher wäre mit der aktuellen Version des Moduls auch nur für max. 3 Ladepunkte möglich. Wäre eine Verbesserung, die ich mir anschaue. Ich würde aber eh noch ein erweitertes Logging einfügen, damit man zur Not auch was nachschauen kann. Wollt ihr mit dem Mergen warten bis ich das getestet habe? Kann allerdings auch ein paar Tage dauern.

Reply:
Ich würde gerne noch die Änderung abwarten.